decentraland/unity-explorer
# style: fix hover/unhover emotes wheel slot animation

## What does this PR change?
This PR was created to fix the unnecessary color transition of emotes wheel's slots when hovering/unhovering them with the cursor. Additionally, the rainbow outline size was diminish in order to avoid overlapping with adjacent slots.

## How to test the changes?
1. Launch the explorer
2. Open the emotes wheel pressing its button or B.
3. Hover the slots an notice the color transition which made them darker is gone and that the rainbow outline is not overlapping adjacent slots anymore.
